ocs_ci/ocs/utils.py

```diff
@@ -1012,3 +1013,24 @@ def revive_osd_external(ceph_cluster, osd_id):
         except CommandFailed:
             log.error("Failed to revive osd")
             raise
+
+
+def reboot_node(ceph_node, timeout=300):
+    """
+    Reboot a node with given ceph_node object
+
+    Args:
+        ceph_node (CephNode): ceph node object representing the node
```

Comment:
```suggestion
        ceph_node (CephNode): Ceph node object representing the node.
```
nitpick but try to be consistent with docstring and if you start one with Capital letter, follow it on all descriptions. And if I am starting with Capital I would also add period at the end of sentence.